src/coreclr/inc/ReferenceOpcodesGenerator/README.md

```diff
@@ -0,0 +1,10 @@
+# Reference opcode generator
+
+Tool to generate the numbering of the reference opcodes for human readability.
```

Comment:
Do we expect that any humans will find the output of this script useful?

This script looks like a left-over from the very early days when the IL instruction was defined and iterated on, and folks wanted to have different visualizations of the current state. It may be better to delete it.